### DISCUSSION TITLE
Nodes stuck 'connecting' after upgrade

### DISCUSSION THREAD

**Discussion by whatdaybob** [zachowj/node-red-contrib-home-assistant-websocket#394] [Q&A]
I have a strange problem after upgrading the version. I upgraded to 0.31.1 and got asked to update my flows with the new nodes but now all my HA nodes are stuck saying 'connecting' and I am not sure why?

![image](https://user-images.githubusercontent.com/5676529/118893908-227f4400-b8fb-11eb-87f1-5305f3609295.png)

**Comment by zachowj** [comment]
How are you running NR? hassos addon/docker/baremetal?

What do the NR logs show?

**Reply by kiddjr73** [reply]
Just updated.  Mine is doing the same thing (hung in connecting).  I am running in docker.

**Reply by kiddjr73** [reply]
I got it working using my duckdns address in the Base URL location.  What I have noticed is the Accept Unauthorized SSL Certificates doesn't stick so I think that was my problem.  Before the update I have always used my internal IP address and that doesn't use the certificates created by LetsEncrypt.  Is the Accept Unauthorized SSL Certificates option the problem?

**Reply by zachowj** [reply]
> What I have noticed is the Accept Unauthorized SSL Certificates doesn't stick so I think that was my problem.

This was one of the issues and the other was a bad migration for the config-server.
